snmp/datadog_checks/snmp/data/conf.yaml.example

```diff
@@ -1,99 +1,109 @@
-  ## configuration, you can directly pass the filename.
-  ## By default, it loads all the profiles installed in the python package and
-  ## in the configuration directory, if the file doesn't start with an underscore.
-  #
-  # profiles:
-  #   f5-big-ip:
-  #     definition_file: f5-big-ip.yaml
-  #   router:
-  #     definition_file: generic-router.yaml
-  #   [...]
-
-  ## @param refresh_oids_cache_interval - integer - optional - default: 0
-  ## (beta feature) Set this option to enable caching of OIDs. The value is the number of seconds before the
-  ## the cache is refreshed.
-  ## This option works best when `oid_batch_size` is set to 128 or 256.
-  ## This is useful to improve performance when fetching metrics from large tables.
-  ## When OID caching is enabled, table row OIDs are reused. This means that the table can be fetched
-  ## using batched GET queries, as opposed to one GETNEXT query per row in the table.
-  ## The caching is disabled if the interval is set to `0` (default).
-  ## Caveat: If some tables rows are dynamic, new rows will only be fetched at every refresh interval.
-  ##         - This means new OIDs will only be picked up when OIDs are refreshed according to the refresh interval.
-  ##         - With auto discovery, if the same IP is assigned to a different device with different OIDs,
-  ##         the OIDs will be refreshed only according to the refresh interval.
-  ## Only available using python SNMP integration
-  #
-  # refresh_oids_cache_interval: 3600
-
-  ## @param optimize_mib_memory_usage - boolean - optional - default: false
-  ## **Experimental** - Setting this flag to true attempts to optimize memory usage related to
-  ## MIB information. This shares internal MIB data across instances.
-  ## Note: The absence of impact on MIB resolution for this strategy is not guaranteed yet.
-  ## Only available using python SNMP integration
-  #
-  # optimize_mib_memory_usage: false
-
-  ## @param mibs_folder - string - optional
-  ## Specify an additional folder for your custom mib files (python format).
-  ## Only available using python SNMP integration
-  #
-  # mibs_folder: /path/to/your/mibs/folder
-
-  ## @param ignore_nonincreasing_oid - boolean - optional - default: false
-  ## Set to true to ignore non increasing oid.
-  ## See https://github.com/etingof/pysnmp/blob/master/docs/source/faq/oids-not-increasing.rst
-  ## Only available using python SNMP integration
-  #
-  # ignore_nonincreasing_oid: false
+    ## @param loader - string - optional - default: python
+    ## Check loader to use for all instances. Available loaders:
+    ## - core: (recommended) Uses new corecheck SNMP integration (faster and lower memory usage)
+    ## - python: Uses the legacy python SNMP integration
+    #
+    # loader: python
+
+    ## @param oid_batch_size - integer - optional - default: 10
+    ## The number of OIDs handled by each batch. Increasing this number improves performance but
+    ## uses more resources.
```

Comment:
Could you add as comment that the default for corecheck is:
- 60 before 7.30
- 5 from >7.30